.github/workflows/lint.yml

```diff
@@ -1,6 +1,6 @@
 name: lint
 
-on: [push, pull_request]
```

Comment:
Let's not trigger the CI for any push, only on pull requests actions (opening, editions, new commits etc.) so that to save some CI time